acceptance/appcharts_test.go

```diff
@@ -0,0 +1,254 @@
+			Expect(out).To(MatchRegexp(`standard *| *Epinio standard deployment`))
+		})
+	})
+
+	Describe("app chart show", func() {
+		It("shows the details of standard app chart", func() {
+			out, err := env.Epinio("", "apps", "chart", "show", "standard")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Show application chart details"))
+			Expect(out).To(MatchRegexp(`Key *| *VALUE`))
+			Expect(out).To(MatchRegexp(`Name *| *standard`))
+			Expect(out).To(MatchRegexp(`Short *| *Epinio standard deployment`))
+			Expect(out).To(MatchRegexp(`Description *| *Epinio standard support chart`))
+			Expect(out).To(MatchRegexp(`for application deployment`))
+			Expect(out).To(MatchRegexp(`Helm Repository *| *|`))
+			Expect(out).To(MatchRegexp(`Helm Chart *| *epinio-application*`))
+		})
+
+		It("fails to show the details of bogus app chart", func() {
+			out, err := env.Epinio("", "apps", "chart", "show", "bogus")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Show application chart details"))
+			Expect(out).To(ContainSubstring("Not Found: Application Chart 'bogus' does not exist"))
+		})
+	})
+
+	Describe("app chart default", func() {
+		AfterEach(func() {
+			// Reset to empty default as the state to be seen at the
+			// beginning of each test, regardless of ordering.
+			out, err := env.Epinio("", "apps", "chart", "default", "")
+			Expect(err).ToNot(HaveOccurred(), out)
+		})
+
+		It("shows nothing by default", func() {
+			out, err := env.Epinio("", "apps", "chart", "default")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Name: not set, system default applies"))
+		})
+
+		It("sets a default", func() {
+			out, err := env.Epinio("", "apps", "chart", "default", "standard")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("New Default Application Chart"))
+			Expect(out).To(ContainSubstring("Name: standard"))
+
+			out, err = env.Epinio("", "apps", "chart", "default")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Name: standard"))
+		})
+
+		It("fails to sets a bogus default", func() {
+			out, err := env.Epinio("", "apps", "chart", "default", "bogus")
+			Expect(err).To(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Not Found: Application Chart 'bogus' does not exist"))
+		})
+
+		It("unsets a default", func() {
+			By("setting default")
+			out, err := env.Epinio("", "apps", "chart", "default", "standard")
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			By("unsetting default")
+			out, err = env.Epinio("", "apps", "chart", "default", "")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Unset Default Application Chart"))
+
+			out, err = env.Epinio("", "apps", "chart", "default")
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(ContainSubstring("Name: not set, system default applies"))
```

Comment:
what is the "system default"? Is that "standard" or something written in the epinio config?

Reply:
https://github.com/epinio/epinio/blob/94b00d0d52d1fc3f52659d12a03ef33dbbbd93b4/internal/api/v1/application/create.go#L91